### PR TITLE
remove duplicate used field from blobber

### PR DESCRIPTION
**Pull request by murthy95**
## Fixes
Removed duplicate `Used` field from blobber.
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/smartcontract/dbs/goose/migrations/1700159742_remove_unused_cols_in_blobber.sql [hunk 1]
```diff
@@ -6,6 +6,7 @@ ALTER TABLE blobbers DROP COLUMN IF EXISTS logo_url;
 ALTER TABLE blobbers DROP COLUMN IF EXISTS description;
 ALTER TABLE blobbers DROP COLUMN IF EXISTS latitude;
 ALTER TABLE blobbers DROP COLUMN IF EXISTS longitude;
+ALTER TABLE blobbers DROP COLUMN IF EXISTS used;
```
Comment (devyetii) [inline code/go/0chain.net/smartcontract/dbs/goose/migrations/1700159742_remove_unused_cols_in_blobber.sql]: Please make a new migration file for this change. Editing migration files may lead to ignoring your new changes if you tried to deploy your changes to a n/w without db cleanup.

**Review by devyetii — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/smartcontract/dbs/goose/migrations/1700159742_remove_unused_cols_in_blobber.sql (hunk 1 above)
Comment (murthy95) [reply code/go/0chain.net/smartcontract/dbs/goose/migrations/1700159742_remove_unused_cols_in_blobber.sql]: Done.

**Review by devyetii — APPROVED** [review]
LGTM

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**